test/openAPI/features/support/g2p_prepayment_validation.js

```diff
@@ -0,0 +1,477 @@
+const chai = require('chai');
+const { spec } = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const {
+  localhost,
+  defaultResponseTime,
+  contentTypeHeader,
+  prepaymentValidationEndpoint,
+  g2pResponseSchema,
+} = require('./helpers/helpers');
+
+chai.use(require('chai-json-schema'));
+
+let specPrePaymentValidation;
+
+const baseUrl = localhost + prepaymentValidationEndpoint;
+const endpointTag = { tags: `@endpoint=/${prepaymentValidationEndpoint}` };
+
+Before(endpointTag, () => {
+  specPrePaymentValidation = spec();
+});
+
+// Scenario: Succesfully retrieves eligible Functional IDs from the account mapper for credit transfers smoke type test
+Given(
+  'Wants to retrieves eligible Functional IDs from the account mapper for credit transfers',
+  () =>
+    'Wants to retrieves eligible Functional IDs from the account mapper for credit transfers'
+);
+
+When(
+  'POST request with required body with given {string} as RequestID, {string} as SourceBBID, {string} as BatchID, {string} as InstructionID, {string} as PayeeFunctionalID, {string} as Amount, {string} as Currency, {string} as Narration is sent',
+  (
+    RequestID,
+    SourceBBID,
+    BatchID,
+    InstructionID,
+    PayeeFunctionalID,
+    Amount,
+    Currency,
+    Narration
+  ) =>
+    specPrePaymentValidation
+      .post(baseUrl)
+      .withHeaders('Accept', 'application/json')
```

Comment:
Please add it to helpers and use it from there, for example: 

  acceptHeader: {
    key: 'Accept',
    value: 'application/json',
  },